pkg/systemlogmonitor/types/types.go

```diff
@@ -42,4 +42,7 @@ type Rule struct {
 	// Pattern is the regular expression to match the problem in log.
 	// Notice that the pattern must match to the end of the line.
 	Pattern string `json:"pattern"`
+	// Message is an optional static message to append to the matching log message.
+	// This can provide additional context or instructions for resolving the issue.
+	Message string `json:"message,omitempty"`
```

Comment:
rename `Message` name to `UserDefinedAdditionalMatchingLogMessage`?

Reply:
maybe `PatternSuffix` or `PatternMessage ` is better ? 🤔 
Because our new field is actually an extended info

Reply:
`PatternGeneratedMessageSuffix`?

Reply:
naming is difficult. :-(

Reply:
also the comment and code should be adjusted.

Reply:
done